sceptre/bridge/config/prod/bridge-ops.yaml

```diff
@@ -0,0 +1,10 @@
+template:
+  type: http
+  url: https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/master/templates/EC2/basic-ec2.j2
+stack_name: bridge-ops
+dependencies:
+  - prod/bootstrap.yaml
```

Comment:
sorry, i missed this.  The VpcId and SubnetId that's passed into this template are created by [BridgeServer2-vpc.yaml](https://github.com/Sage-Bionetworks-IT/organizations-infra/blob/master/sceptre/bridge/config/develop/BridgeServer2-vpc.yaml) so probably should replace this with `prod/BridgeServer2-vpc.yaml`